Libraries/WebSocket/WebSocket.js

```diff
@@ -24,15 +24,16 @@ var CLOSE_NORMAL = 1000;
  * Browser-compatible WebSockets implementation.
  *
  * See https://developer.mozilla.org/en-US/docs/Web/API/WebSocket
+ * See https://github.com/websockets/ws
  */
 class WebSocket extends WebSocketBase {
   _socketId: number;
   _subs: any;
 
-  connectToSocketImpl(url: string): void {
+  connectToSocketImpl(url: string, protocols: ?any, options: ?any): void {
```

Comment:
Why annotate with `?any`? Isn't it better to have `protocols: ?Array<string>, options: ?Object`?